wordless/theme_builder/vanilla_theme/webpack.env.js

```diff
@@ -6,7 +6,7 @@ devOptions = {
     {
       loader: MiniCssExtractPlugin.loader,
       options: {
-        sourceMap: true
```

Comment:
why did you remove the `sourceMap` key? is it moved somewhere else? given this update is neither necessary to put the configuration inside `.env.js` part of the configuration, since they are equal for both envoronment.
But I think that switching on/off by environment the generation of source maps was intended as a feature.

Reply:
Because,

ERROR in ./src/stylesheets/screen.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ValidationError: Invalid options object. Mini CSS Extract Plugin Loader has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'sourceMap'. These properties are valid:
 
 and I didn't found a solution yet

Reply:
As discussed elsewhere, sourcemap production can be controlled directly through `devtools` key, so it's just ok to have removed this config line